docs/2/technical/data-format/overview.md

```diff
@@ -114,55 +107,49 @@ Here we will see one by one the different components, starting with the unique v
 
 * `"creatorId"`
 
-CreatorId indicates who created this piece of data, you will find here the email of the account that created this data.
+CreatorId suplies the email address of the creator this piece of data.
 
 * `"desc"`
 
-This value is usually empty, so you can ignore it when it is empty.
-
-* `"id"`
-
-This value is exactly the same as `"_id"`
+This object serves to describe the sensor the information comes from. Usually, this object is empty. 
 
 * `"name"`
 
-This value is usually empty, so you can ignore it when it is empty.
+This object gives the name of the sensor that the information comes from. This field is usually empty. 
 
 * `"recType"`
 
-If you are on this page it's because you have started to manipulate the Microshare® APIs and in this case you are familiar with the notion of recType, as a quick reminder the recType is the endpoint of a data flow in the data lake.
+The recType is the endpoint of a data flow in the data lake, the name underwhich your data is stored. [You can learn more about recTypes here.](/docs/2/technical/api/api-collection/#api-standards)
 
 * `"tstamp"`
 
-This is the timestamp (in milliseconds), here is more information and a site to test this: 
-
-The Unix epoch (or Unix time or POSIX time or Unix timestamp) is the number of seconds that have elapsed since January 1, 1970 (midnight UTC/GMT), not counting leap seconds (in ISO 8601: 1970-01-01T00:00:00Z). Literally speaking the epoch is Unix time 0 (midnight 1/1/1970), but 'epoch' is often used as a synonym for Unix time. Some systems store epoch dates as a signed 32-bit integer, which might cause problems on January 19, 2038 (known as the Year 2038 problem or Y2038). The converter on this page converts timestamps in seconds (10-digit), milliseconds (13-digit) and microseconds (16-digit) to readable dates.
+This is the timestamp (in milliseconds)in the Unix epoch unit. The Unix epoch (or Unix time or POSIX time or Unix timestamp) is the number of seconds that have elapsed since January 1, 1970 (midnight UTC/GMT), not counting leap seconds (in ISO 8601: 1970-01-01T00:00:00Z). Literally speaking the epoch is Unix time 0 (midnight 1/1/1970), but 'epoch' is often used as a synonym for Unix time. Some systems store epoch dates as a signed 32-bit integer, which might cause problems on January 19, 2038 (known as the Year 2038 problem or Y2038). The converter on this page converts timestamps in seconds (10-digit), milliseconds (13-digit) and microseconds (16-digit) to readable dates.
 
 [https://www.epochconverter.com/](https://www.epochconverter.com/)
 
 Here the value is 1590759984779 so the date is Friday 29 May 2020 13:46:24.779 which is exactly the `createDate` value : "2020-05-29T13:46:24.779Z"
 
 * `"updateDate"`
 
-Here the data has not been updated, so the `updateDate` is the same as the `createDate`
+The `updateDate` gives the time that the data was updated. Here the data has not been updated, so the `updateDate` is the same as the `createDate`
```

Comment:
UpdateDate is an ISO 8601 date/time stamp in the YYYY-MM-DDThh:mm:ss.sTZD pattern that represents the date/time of the data's last update in the Microshare system. If the data has not been updated, it will be the same as the createDate.